src/VirtualClient/VirtualClient.Dependencies/CUDAAndNvidiaGPUDriverInstallation.cs

```diff
@@ -105,6 +122,41 @@ public string Username
             }
         }
 
+        /// <summary>
+        /// The Cuda Toolkit Windows executable Command Line Arguments.
+        /// </summary>
+        public string WinCommandLineArgs
+        {
+            get
+            {
+                return this.Parameters.GetValue<string>(nameof(CudaAndNvidiaGPUDriverInstallation.WinCommandLineArgs), "-y -s");
+            }
+        }
+
+        /// <summary>
+        /// If this is true, VC expects CUDA and NVIDIA Drivers Installer Package from Blob Storage.
+        /// If false, CUDA and NVIDIA Drivers installer will be downloaded from Web based on link provided.
+        /// </summary>
+        public bool WinCudaDriversPackageDownloadedFromBlob
```

Comment:
Do we need this? I'm not seeing this in the profile

Reply:
There are two ways by which the driver download will be supported, one is through the Web (which I have put in the profile), and the other is blob. The blob part should prove helpful in either disconnected scenarios or when GPU vendors share private drivers that are to be used for NPI skus, they can be uploaded in blob and used. I'll add the use-case in the documentation for this dependency.